app/services/public_safety_service.rb

```diff
@@ -0,0 +1,57 @@
+#
+# Service class pattern based on Engine Yard example:
+# http://www.engineyard.com/blog/keeping-your-rails-controllers-dry-with-services
+#
+class PublicSafetyService
+  # Static/Class Methods
+  class << self
+    # This is the title the Chapman alert feed is expected to have if there is no
+    # emergency.
+    # TODO: Check with Public Safety to see if this message if machine or user
+    # generated. If user-generated, this is dangerous since it means user inputs
+    # it once the emergency is complete and if they don't input it as expected,
+    # signs would continue to display emergency alert.
+    NO_EMERGENCY_TITLE = 'There is currently no emergency.'.freeze
```

Comment:
This is the part that makes me nervous. I would prefer emergencies were more opt-in in flavor rather than opt-out as they are now.